ui/src/apps/overview/RootComponent.js

```diff
@@ -40,9 +64,28 @@ const App = () => {
               <Card
                 size="small"
                 bordered={false}
-                title={t('overview.top_statements.title')}
+                title={
+                  <div style={{ display: 'flex', alignItems: 'baseline' }}>
+                    <h3 style={{ marginRight: 8 }}>
+                      {t('overview.top_statements.title')}
```

Comment:
We can also display the time range here.

Reply:
Good point, will add it soon.